pkg/jx/cmd/install.go

```diff
@@ -923,321 +909,815 @@ func (options *InstallOptions) Run() error {
 			options.Flags.Prow = true
 		}
 	}
+	return nil
+}
 
-	log.Infof("Installing Jenkins X platform helm chart from: %s\n", makefileDir)
-
-	options.Verbose = true
-	err = options.addHelmBinaryRepoIfMissing(DEFAULT_CHARTMUSEUM_URL, "jenkins-x")
-	if err != nil {
-		return errors.Wrap(err, "failed to add the jenkinx-x helm repo")
+func (options *InstallOptions) configureTillerNamespace() error {
+	helmConfig := &options.CreateEnvOptions.HelmValuesConfig
+	initOpts := &options.InitOptions
+	if initOpts.Flags.TillerNamespace != "" {
+		if helmConfig.Jenkins.Servers.Global.EnvVars == nil {
+			helmConfig.Jenkins.Servers.Global.EnvVars = map[string]string{}
+		}
+		helmConfig.Jenkins.Servers.Global.EnvVars["TILLER_NAMESPACE"] = initOpts.Flags.TillerNamespace
+		os.Setenv("TILLER_NAMESPACE", initOpts.Flags.TillerNamespace)
 	}
+	return nil
+}
 
-	version := options.Flags.Version
-	if version == "" {
-		version, err = LoadVersionFromCloudEnvironmentsDir(wrkDir, configStore)
-		if err != nil {
-			return errors.Wrap(err, "failed to load version from cloud environments dir")
-		}
+func (options *InstallOptions) configureHelmValues(namespace string) error {
+	helmConfig := &options.CreateEnvOptions.HelmValuesConfig
+	if helmConfig.ExposeController.Config.Domain == "" {
+		helmConfig.ExposeController.Config.Domain = options.InitOptions.Flags.Domain
 	}
 
-	err = options.Helm().UpdateRepo()
-	if err != nil {
-		return errors.Wrap(err, "failed to update the helm repo")
+	domain := helmConfig.ExposeController.Config.Domain
+	if domain != "" && addon.IsAddonEnabled("gitea") {
+		helmConfig.Jenkins.Servers.GetOrCreateFirstGitea().Url = "http://gitea-gitea." + namespace + "." + domain
```

Comment:
why `gitea-gitea` and not just `gitea`?

Reply:
This is the original code. I don't want to modify in the PR. It can be refactoedr later.